### ISSUE TITLE
Bring back me for changing image

### ISSUE THREAD

**Issue by vuzDylan** [rit-sse/OneRepoToRuleThemAll#34]

**Comment by kristenmills** [comment]
Hmm good point

**Comment by vuzDylan** [comment]
Almost done with this just need to update the node api to allow users to update themselves

**Comment by billdybas** [comment]
Yes. See my comment in #141 for an idea about this: dashboard for users w/ relevant information to them w/ a form to update their display info.
